var/spack/repos/builtin/packages/neurodamus-thalamus/package.py

```diff
@@ -12,5 +12,6 @@ class NeurodamusThalamus(NeurodamusModel):
     git      = "ssh://bbpcode.epfl.ch/sim/models/thalamus"
 
     version('develop', git=git, branch='master', submodules=True, clean=False)
+    version('0.2.1', git=git, tag='0.2', submodules=True, clean=False)
     version('0.2', git=git, tag='0.2', submodules=True, clean=False)
```

Comment:
In `deploy/configs/applications/modules.yaml` we should only whitelist latest version to avoid all modules?

Reply:
Made the required changes in `bbp-packages.yaml`.

Reply:
> In `deploy/configs/applications/modules.yaml` we should only whitelist latest version to avoid all modules?

No. We should whitelist only the desired packages